compiler/noirc_frontend/src/tests/turbofish.rs

```diff
@@ -45,6 +45,8 @@ fn turbofish_numeric_generic_nested_method_call() {
 
     fn bar<let N: u32>() -> [u8; N] {
         let _ = Foo::static_method::<N>();
+                     ^^^^^^^^^^^^^ Type annotation needed
+                     ~~~~~~~~~~~~~ Could not determine the type of the generic argument `T` declared on the struct `Foo`
```

Comment:
```suggestion
        let _ = Foo::<u8>::static_method::<N>();
```

I think we want to supply `T` rather than asserting an error here.

Reply:
You are right. However, it still doesn't compile for some reason so I'll try to figure out why...

Reply:
That said, the code doesn't compile in master either, so the bug is unrelated to this PR. Should I try to fix it in this PR?